website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -116,27 +120,30 @@ To upgrade from Community Edition to Business Edition, follow the instructions a
 
 4. Add the Appsmith chart repository with:
 
-```bash
-helm repo add appsmith-ee https://helm-ee.appsmith.com
-```
+  ```bash
+  helm repo add appsmith-ee https://helm-ee.appsmith.com
+  ```
 5. Load the Appsmith chart repository with:
 
-```bash
-helm repo update
-```
+  ```bash
+  helm repo update
+  ```
 6. Run the below command to deploy Appsmith:
 
-```bash
-helm install appsmith appsmith-ee/appsmith
-```
+  ```bash
+  helm install appsmith appsmith-ee/appsmith
```

Comment:
Incase we want users to create a namespace and isolate things from
we can suggest the following 
```
helm install appsmith appsmith/appsmith -n appsmith --create-namespace
```

and change this from
```
kubectl --namespace default port-forward <APPSMITH_POD_NAME> 8080:80
```

to
```
kubectl --namespace appsmith port-forward <APPSMITH_POD_NAME> 8080:80
```